### ISSUE TITLE
Expose types

### ISSUE THREAD

**Issue by PeterBurner** [serverless-guru/prettier-import-order#5]
**Is your feature request related to a problem?**
I would like to be able to use `JSDoc` type information at `.prettierrc.js` for better IDE support (auto completion etc.).

**Describe the solution you'd like**
Could you please add TypeScript typings to the npm release? You already have a `types.ts` file.
Example:
```js
/**
 * @type {import('@serverless-guru/prettier-plugin-import-order').PrettierOptions}
 */
module.exports = {
  useTabs: false,
  semi: true,
  singleQuote: true,
  trailingComma: 'none',
  bracketSpacing: true,
  arrowParens: 'always',
  importOrder: [
    '^dotenv(.*)$',
    '<THIRD_PARTY_MODULES>',
    '^@aws-sdk/(.*)$',
    '^[./]'
  ],
  importOrderSortIndividualImports: true,
  importOrderMergeDuplicateImports: true,
  importOrderSeparation: true,
  importOrderSortSpecifiers: true
};

```

**Describe alternatives you've considered**
Currently I am using the official types of `prettier` which are missing `importOrder` and so on.
```js
/**
 * @type {import('prettier').Options}
 */
module.exports = {
  useTabs: false,
  semi: true,
  singleQuote: true,
  trailingComma: 'none',
  bracketSpacing: true,
  arrowParens: 'always',
  importOrder: [
    '^dotenv(.*)$',
    '<THIRD_PARTY_MODULES>',
    '^@aws-sdk/(.*)$',
    '^[./]'
  ],
  importOrderSortIndividualImports: true,
  importOrderMergeDuplicateImports: true,
  importOrderSeparation: true,
  importOrderSortSpecifiers: true
};
```

**Additional context**
 - https://github.com/serverless-guru/prettier-import-order/blob/main/src/types.ts#L13
 - https://jsdoc.app/tags-type.html

**Comment by tushar-sg** [comment]
Hi @PeterBurner I will work on it this week. Thanks for submitting the issue.

**Comment by tusharf5** [comment]
@PeterBurner The types should now be available in 0.4.1

**Comment by PeterBurner** [comment]
Works. Thanks for the effort!
